<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TInterface</class>
 <widget class="QMainWindow" name="TInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OOP Coursework (Bouncing ball)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="TCanvas" name="canvas" native="true"/>
    </item>
    <item>
     <spacer name="spacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lwLabel">
        <property name="text">
         <string>L x W:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEdit">
        <property name="text">
         <string>800</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="wEdit">
        <property name="text">
         <string>600</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rLabel">
        <property name="text">
         <string>R:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rEdit">
        <property name="text">
         <string>25</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vLabel">
        <property name="text">
         <string>V:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="vEdit">
        <property name="text">
         <string>20</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="aLabel">
        <property name="text">
         <string>A:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="aEdit">
        <property name="text">
         <string>45</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="applyBtn">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stateBtn">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TCanvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
